app/services/proofing/aamva/proofer_factory.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Proofing
+  module Aamva
+    class ProoferFactory
+      attr_accessor :resolution_context
+
+      # @param [Proofing::Resolution::ResolutionContext] resolution_context
+      def initialize(resolution_context)
+        @resolution_context = resolution_context
+      end
+
+      def get_proofer
```

Comment:
`get_proofer` is the primary public method for this class, so its params should be documented. (same applies to other proofer factory)